proto/osmosis/concentrated-liquidity/gov.proto

```diff
@@ -5,30 +5,18 @@ import "gogoproto/gogo.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/types";
 
-// CreateConcentratedLiquidityPoolProposal is a gov Content type for creating a
-// concentrated liquidity pool. If a CreateConcentratedLiquidityPoolProposal
-// passes, pool is created via pool manager module account.
-message CreateConcentratedLiquidityPoolProposal {
+// CreateConcentratedLiquidityPoolsProposal is a gov Content type for creating
+// concentrated liquidity pools. If a CreateConcentratedLiquidityPoolsProposal
+// passes, the pools are created via pool manager module account.
+message CreateConcentratedLiquidityPoolsProposal {
   option (gogoproto.equal) = true;
   option (gogoproto.goproto_getters) = false;
   option (gogoproto.goproto_stringer) = false;
 
   string title = 1;
   string description = 2;
 
-  string denom0 = 3 [ (gogoproto.moretags) = "yaml:\"denom0\"" ];
-  string denom1 = 4 [ (gogoproto.moretags) = "yaml:\"denom1\"" ];
-  uint64 tick_spacing = 5 [ (gogoproto.moretags) = "yaml:\"tick_spacing\"" ];
-  string exponent_at_price_one = 6 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.moretags) = "yaml:\"exponent_at_price_one\"",
-    (gogoproto.nullable) = false
-  ];
-  string spread_factor = 7 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
-    (gogoproto.moretags) = "yaml:\"spread_factor\"",
-    (gogoproto.nullable) = false
-  ];
+  repeated PoolRecord pool_records = 3 [ (gogoproto.nullable) = false ];
```

Comment:
Should we be adding a tag here? e.g  `(gogoproto.moretags) = "yaml:\"pool_records\""`